.script/dataConnectorValidator.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "fs";
+import { runCheckOverChangedFiles } from "./utils/changedFilesValidator";
+import { ExitCode } from "./utils/exitCode";
+import { isValidSchema } from "./utils/jsonSchemaChecker";
+import { isValidId } from "./utils/dataConnectorCheckers/idChecker";
+import { isValidDataType } from "./utils/dataConnectorCheckers/dataTypeChecker";
+import * as logger from "./utils/logger";
+
+export async function IsValidDataConnectorSchema(filePath: string): Promise<ExitCode> {
+  let dataConnector = JSON.parse(fs.readFileSync(filePath, "utf8"));
```

Comment:
variable name is misleading since it may not be a connector json. rename to generic jsonFile.